runtime/src/lib.rs

```diff
@@ -95,7 +95,7 @@ impl sdk::Runtime for Runtime {
     );
 
     fn trusted_policy_signers() -> Option<TrustedPolicySigners> {
-        if option_env!("OASIS_UNSAFE_SKIP_KM_POLICY") == Some("1") {
+        if option_env!("OASIS_UNSAFE_SKIP_KM_POLICY").is_some() {
```

Comment:
strictly speaking, I think it's the other code that's not ideal. `OASIS_UNSAFE_SKIP_KM_POLICY=no` seems like not a great way to turn it on.

Reply:
Besides being consistent it also has the side effect of shutting up clippy. Let me add an ignore instead.